pkgs/development/interpreters/python/hooks/default.nix

```diff
@@ -122,6 +122,12 @@ in rec {
       };
     } ./python-remove-tests-dir-hook.sh) {};
 
+  pythonWrapExecutablesHook = callPackage ({ wrapPython }:
```

Comment:
This hook would replace `wrapPython`, so users typically do not need to invoke `wrapPythonPrograms(In)` anymore. cc @jonringer